cache-config/testing/docker/traffic_ops/run.sh

```diff
@@ -149,6 +149,7 @@ cd /opt/traffic_ops/app
 (
 db/admin --env=production reset
 db/admin --env=production upgrade
+db/admin --env=production seed
```

Comment:
Why should seeding happen after running migrations? Isn't the whole point of no longer seeding when running migrations that `seeds.sql` only changes when the DB is squashed?

If that's the case, it should run before migrations.

Reply:
No, the point is to not overwrite changes users have made to their system's data. I don't think it really matters which order it's done in, but it does seem to me to be more appropriate to copy the way `postinstall` does it, which is
1. load_schema
2. migrate
3. seed
4. patch

Reply:
> but it does seem to me to be more appropriate to copy the way `postinstall` does it

The `postinstall` *did* it. Nothing in postinstall runs seeds anymore except for explicitly seeding. That could mess up people who don't use the postinstall script, it makes the process more manual

Reply:
Yes, that's correct, the postinstall script seeds manually. Which is fortunate because after this change that's the correct way to invoke it. Not sure what you mean by "did" - that is still the current default behavior.